libs/core/src/lib/tabs/tab-panel/tab-panel.component.ts

```diff
@@ -134,6 +141,17 @@ export class TabPanelComponent implements OnChanges {
             }
 
             this._changeDetRef.detectChanges();
+
+            // Done this way to avoid "NG0100: Expression has changed after it was checked"
+            setTimeout(() => {
+                this._updateHost();
+            });
         }
     }
+
+    /** @hidden */
+    private _updateHost(): void {
+        this._expandedClass = this.expanded;
+        this._expandedAria = this.expanded ? true : null;
```

Comment:
why `null` but not `false`?

Reply:
when binding to attribute null evaluates to "no attribute at all", when false only about setting value.